neqo-transport/src/connection.rs

```diff
@@ -2514,7 +2514,7 @@ mod tests {
         assert_eq!(0, client.stats().dups_rx);
 
         qdebug!("---- Dup, ignored");
-        let out = client.process(out_to_rep.dgram().clone(), now());
```

Comment:
I totally get this one: the dgram() function returns an object directly, so cloning it is wasteful.  But see below.